Tests/UnitTests/Plugins/Json/PortalJsonConverterTests.cpp

```diff
@@ -0,0 +1,114 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2023 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <boost/test/unit_test.hpp>
+
+#include "Acts/Definitions/Algebra.hpp"
+#include "Acts/Definitions/Direction.hpp"
+#include "Acts/Detector/Portal.hpp"
+#include "Acts/Detector/detail/PortalHelper.hpp"
+#include "Acts/Geometry/GeometryContext.hpp"
+#include "Acts/Plugins/Json/PortalJsonConverter.hpp"
+#include "Acts/Surfaces/PlaneSurface.hpp"
+#include "Acts/Surfaces/Surface.hpp"
+#include "Acts/Tests/CommonHelpers/FloatComparisons.hpp"
+
+#include <fstream>
+
+#include <nlohmann/json.hpp>
+
+namespace Acts {
+namespace Experimental {
+class DetectorVolume {};
+}  // namespace Experimental
+}  // namespace Acts
+
+Acts::GeometryContext tContext;
+
+BOOST_AUTO_TEST_SUITE(PortalJsonConverter)
+
+BOOST_AUTO_TEST_CASE(PortalUnconnected) {
```

Comment:
These test don't read anything back in to test the ```fromJson``` or will that be added later?

Reply:
Tests are added.